From ab8f035e4510df70297a306e8b75309705f4b588 Mon Sep 17 00:00:00 2001
From: DevUt <javitri61@gmail.com>
Date: Fri, 24 Jun 2016 18:48:43 +0530
Subject: [PATCH] Update init.rc

---
 rootdir/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index ec5a2f6..aab8f09 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -6,9 +6,9 @@
 
 import /init.environ.rc
 import /init.usb.rc
-import /init.${ro.hardware}.rc
+import /init.endeavoru.rc
 import /init.usb.configfs.rc
-import /init.${ro.zygote}.rc
+import /init.zygote32.rc
 import /init.trace.rc
 # Include CM's extra init file
 import /init.cm.rc