From 49db81af0dac08b91c40f0b7b7727d11e7537a3e Mon Sep 17 00:00:00 2001
From: alfsamsung <alfsamsung@hotmail.com>
Date: Mon, 9 Feb 2015 16:52:43 +0100
Subject: [PATCH] Remove some logging

Signed-off-by: alfsamsung <alfsamsung@hotmail.com>
---
 linker/linker.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index 8e71648..716bd65 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -3066,8 +3066,10 @@ bool soinfo::link_image(const soinfo_list_t& global_group, const soinfo_list_t&
     }
     // Make segments writable to allow text relocations to work properly. We will later call
     // phdr_table_protect_segments() after all of them are applied and all constructors are run.
+#if !defined(STOP_LOG_SPAM)
     DL_WARN("%s has text relocations. This is wasting memory and prevents "
             "security hardening. Please fix.", get_realpath());
+#endif
     if (phdr_table_unprotect_segments(phdr, phnum, load_bias) < 0) {
       DL_ERR("can't unprotect loadable segments for \"%s\": %s",
              get_realpath(), strerror(errno));
-- 
1.9.1
